exporter/exporterhelper/queue_sender.go

```diff
@@ -211,17 +211,6 @@ func (qs *queueSender) Shutdown(ctx context.Context) error {
 
 // send implements the requestSender interface
 func (qs *queueSender) send(req internal.Request) error {
-	if qs.queue == nil {
```

Comment:
Not sure if there a bug to be fixed. retry sender has a similar check https://github.com/open-telemetry/opentelemetry-collector/blob/23500dda16b565d5f15c535e5a9d22927e4700d2/exporter/exporterhelper/retry_sender.go#L110

Reply:
If that's the case, can you please remove this block ^ and the change log entry

Reply:
Correct.